src/main.cpp

```diff
@@ -1411,7 +1411,7 @@ bool ContextualCheckShieldedInputs(
 
     // Queue Orchard bundle to be batch-validated.
     if (orchardAuth.has_value()) {
-        tx.GetOrchardBundle().QueueAuthValidation(orchardAuth.value(), dataToBeSigned);
+        tx.GetOrchardBundle().QueueAuthValidation(*orchardAuth.value(), dataToBeSigned);
```

Comment:
Use parens to make the precedence clearer (since both `*` and `.value()` apply to a `std::optional`, with different meanings wrt error behaviour).